source/module_cell/cell_index.h

```diff
@@ -0,0 +1,65 @@
+#ifndef CELL_INDEX_H
```

Comment:
I am not sure what is this class used for? There is no clue in issue #4170 indicating this change.

Reply:
In the process of refactoring, the need of a class to manage cell indices arose. So I added this class to be used in Output_Mulliken. It is a simple class but can provide many indices info, and I think it might be helpful in refactoring UnitCell as it can be used to replace GlobalC::ucell when only cell indices are needed. For example, the function ofwrite_orb_info has been added as a member function of CellIndex.